joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -554,6 +555,18 @@ rclcpp_action::GoalResponse JointTrajectoryController::goal_callback(
     }
   }
 
+  for (auto i = 0ul; i < goal->trajectory.joint_names.size(); ++i) {
+    std::string incoming_joint_name = goal->trajectory.joint_names[i];
```

Comment:
```suggestion
    const std::string& incoming_joint_name = goal->trajectory.joint_names[i];
```
to avoid creating a copy

Reply:
but we can put this on a follow-up ticket if you want

Reply:
I've fixed it up on your branch, waiting for CI to pass